fakeData.js

```diff
@@ -0,0 +1,33 @@
+const faker = require('faker');
+const fs = require('fs');
+const menuItems = require('./helpers/menuGenerator');
+const file = fs.createWriteStream(`./output.json`);
+
+function writeOneMillionTimes(file, encoding) {
```

Comment:
you could make your function take `i` as a parameter and name it so it is more general than `writeOneMillionTimes`.  Not really a big deal since I don't think this will be called often